_oasis

```diff
@@ -28,7 +28,6 @@ Library networklibs
     xcp-inventory,
     xcp.network,
     systemd
-  CSources:           link_stubs.c, netdev.h
   CCOpt:              -Wno-unused-function -g -ggdb
```

Comment:
The `CCOpt` line is harmless, but I'm not sure whether or not it would be better to remove it.